server.go

```diff
@@ -1241,6 +1243,16 @@ out:
 	s.wg.Done()
 }
 
+// Services returns the services that the server supports.  This is safe for
+// concurrent access.
+func (s *server) Services() wire.ServiceFlag {
```

Comment:
As mentioned elsewhere, the server services don't need to be protected by a mutex since they are not mutated after initial creation.  That means this entire function can go away.